web/src/components/dropdowns/query-options-panel.tsx

```diff
@@ -202,33 +201,31 @@ export const QueryOptionsPanel: React.FC<QueryOptionsProps> = ({
           );
         })}
       </div>
-      {deduperMark && (
-        <div className="pf-v5-c-menu__group">
-          <Tooltip
-            content={t(
-              // eslint-disable-next-line max-len
-              'A flow might be reported from several interfaces, and from both source and destination nodes, making it appear several times. By default, duplicates are hidden. Showing duplicates is not possible in Overview and Topology tabs to avoid altering metric calculations. Use the Direction filter to switch between ingress, egress and inner-node traffic.'
-            )}
-          >
-            <div className="pf-v5-c-menu__group-title">
-              <Text component={TextVariants.p}>
-                {t('Duplicated flows')} <InfoAltIcon />
-              </Text>
-            </div>
-          </Tooltip>
-          <label className="display-dropdown-padding pf-v5-c-menu__menu-item">
-            <Checkbox
-              isChecked={allowShowDuplicates ? showDuplicates : false}
-              isDisabled={!allowShowDuplicates}
-              name={'show-duplicates'}
-              onChange={() => setShowDuplicates(!showDuplicates)}
-              label={t('Show duplicates')}
-              data-test={'show-duplicates'}
-              id={'show-duplicates'}
-            />
-          </label>
-        </div>
-      )}
+      <div className="pf-v5-c-menu__group">
+        <Tooltip
+          content={t(
+            // eslint-disable-next-line max-len
+            'A flow might be reported from several interfaces, and from both source and destination nodes, making it appear several times. By default, duplicates are hidden. Showing duplicates is not possible in Overview and Topology tabs to avoid altering metric calculations. Use the Direction filter to switch between ingress, egress and inner-node traffic.'
+          )}
+        >
+          <div className="pf-v5-c-menu__group-title">
+            <Text component={TextVariants.p}>
+              {t('Duplicated flows')} <InfoAltIcon />
+            </Text>
+          </div>
+        </Tooltip>
+        <label className="display-dropdown-padding pf-v5-c-menu__menu-item">
+          <Checkbox
+            isChecked={allowShowDuplicates ? showDuplicates : false}
+            isDisabled={!allowShowDuplicates}
+            name={'show-duplicates'}
+            onChange={() => setShowDuplicates(!showDuplicates)}
+            label={t('Show duplicates')}
+            data-test={'show-duplicates'}
+            id={'show-duplicates'}
+          />
+        </label>
+      </div>
```

Comment:
Should we keep that in the query options or move it in the table display options ?

Reply:
done: 
![image](https://github.com/user-attachments/assets/25f01119-8294-4074-ab32-b6788281621c)